file-formats/edcc/type/zif_aff_edcc_v1.intf.abap

```diff
@@ -0,0 +1,588 @@
+      "! Assigned status checks
+      status_check         TYPE ty_status_checks,
+      "! <p class="shorttext">Content Checks</p>
+      "! Assigned content checks
+      content_check        TYPE ty_content_checks,
+      "! <p class="shorttext">Check Attributes</p>
+      "! Check attributes
+      "! $required
+      check_attributes     TYPE ty_check_attributes,
+      "! <p class="shorttext">Representation Types</p>
+      "! Assigned representation types
+      "! $required
+      representation_types TYPE ty_representation_types,
+    END OF ty_comparison_type.
+  "! <p class="shorttext">Comparison Types</p>
+  "! Define comparison types
+  TYPES ty_comparison_types TYPE SORTED TABLE OF ty_comparison_type WITH UNIQUE KEY comparison_type.
+  "! <p class="shorttext">Relevance</p>
+  "! Relevance
+  "! $values {@link zif_aff_edcc_v1.data:co_relevance}
+  "! $default {@link zif_aff_edcc_v1.data:co_relevance.unchanged}
+  TYPES ty_relevance        TYPE c LENGTH 1.
+  "! <p class="shorttext">Event Name</p>
+  "! Event name
+  TYPES ty_event_name       TYPE c LENGTH 25.
+  TYPES:
+    "! <p class="shorttext">Assigned Comparison</p>
+    "! Assigned comparison
+    BEGIN OF ty_assigned_comparison,
+      "! <p class="shorttext">Comparison Type</p>
+      "! Comparison type
+      comparison_type TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Check Id</p>
+      "! Check id
+      check_id        TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_assigned_comparison.
+  "! <p class="shorttext">Assign Checks and Comparison Type</p>
+  "! Assign Checks and Comparison Type
+  TYPES ty_assigned_comparisons TYPE SORTED TABLE OF ty_assigned_comparison WITH UNIQUE KEY
+        comparison_type check_id.
+  TYPES:
+    "! <p class="shorttext">Check Relevance</p>
+    "! Define relevance of check
+    BEGIN OF ty_check_relevance,
+      "! <p class="shorttext">Comparison Type</p>
+      "! Comparison type
+      "! $required
+      comparison_type TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Check Id</p>
+      "! Check id
+      "! $required
+      check_id        TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Relevance</p>
+      "! Relevance
+      relevance       TYPE ty_relevance,
+    END OF ty_check_relevance.
+  "! <p class="shorttext">Check Relevance</p>
+  "! Define relevance of checks
+  TYPES ty_check_relevances TYPE SORTED TABLE OF ty_check_relevance WITH UNIQUE KEY
+                              comparison_type check_id.
+  TYPES:
+    "! <p class="shorttext">Assign Event</p>
+    "! Assign event to consistency scenario
+    BEGIN OF ty_event,
+      "! <p class="shorttext">Event Name</p>
+      "! Event name
+      event               TYPE ty_event_name,
+      "! <p class="shorttext">Assign Checks and Comparison Type</p>
+      "! Assign Checks and Comparison Type
+      assigned_comparison TYPE ty_assigned_comparisons,
```

Comment:
```suggestion
      "! <p class="shorttext">Checks and Comparison Types</p>
      "! Assign Checks and Comparison Type
      checks_and_comparison_types TYPE ty_assigned_comparisons,
```

Reply:
Besides that, this seems to be again kind of double information. Checks and Comparison Types have been there before. 
Each check belongs to a comparison type. Why do we need this here again?